Scripts/Core/EditorVR.Tools.cs

```diff
@@ -164,6 +164,10 @@ ToolData SpawnTool(Type toolType, out HashSet<InputDevice> usedDevices, InputDev
 					usedDevices.UnionWith(actionMapInput.GetCurrentlyUsedDevices());
 					InputUtils.CollectDeviceSlotsFromActionMapInput(actionMapInput, ref deviceSlots);
 				}
+				else
```

Comment:
Not quite. The logic changed. It still needs to be a separate `if (usedDevices.Count == 0)` because it's possible that even with an action map assigned that no devices came through from the action map (I think this happened for one IExclusiveMode tool).